services/src/main/java/com/linkedin/kmf/services/TopicManagementService.java

```diff
@@ -62,20 +63,22 @@
     final Set<Integer> _electedLeaders;
     final Collection<Broker> _allBrokers;
     final double _partitionToBrokerRatioThreshold;
+    final int _topicReplicationFactor;
```

Comment:
`_topicReplicationFactor` should not be part of the `TopicState` because it is statically configured and will have the same value for all `TopicState` instances. It should be put in the same place as `_topicCreationEnabled`.

Besides, why rename it from `_topicReplicationFactor` to `_configuredReplicationFactor`?